src/main/java/nl/tudelft/semgroup4/util/QuadTree.java

```diff
@@ -136,6 +136,41 @@ private int getIndex(Shape rect) {
 
         return index;
     }
+    
+    private List<Integer> getIndices(Shape rect) {
```

Comment:
This method is rather long, maybe split it into smaller pieces?

< layout, organisation